src/AbstractPhpdocToTypeDeclarationFixer.php

```diff
@@ -168,6 +168,9 @@ protected function createTypeDeclarationTokens(string $type, bool $isNullable):
         return $newTokens;
     }
 
+    /**
+     * @return null|array{string, bool}
```

Comment:
what is the meaning of string? what is the meaning of boolean value?

Reply:
`[$commonType, $isNullable]`, where `$commonType` is a docblock type simplified to what can be a native type hint, eg. a common type for `Traversable<string>` would be `Traversable`.

Reply:
i'm wondering that if we start documenting the return, we can tell what is actual string or bool here, without asking user to read the codebase

Reply:
It would be nice to have it documented, but the point of this PR is to reduce the PHPStan baseline.